primitives/api-client-extensions/Cargo.toml

```diff
@@ -5,12 +5,12 @@ authors = ["clangenbacher <christian.langenbacher@scs.ch>"]
 edition = "2018"
 
 [dependencies]
-codec           = { package = "parity-scale-codec", version = "2.0.0", features = ["derive"] }
+codec = { package = "parity-scale-codec", version = "2.0.0", features = ["derive"] }
```

Comment:
So we don't want the dependencies to be tab aligned?

Reply:
I don't think that makes sense, because it's wasted horizontal space and does not improve readability for packages that do not import the same features. @clangenb what do you think?

Reply:
Hmm, I also don't think that I want them to be tab-aligned. We used to do that in the very beginning, and there are still some remnants of it. Overall, readability is not improved IMO. On the contrary, you might lose the correct line.